cmd/localCI/github.go

```diff
@@ -248,20 +255,30 @@ func (g *Github) downloadPullRequest(pr int, workingDirectory string) (string, e
 
 	var stderr bytes.Buffer
 
+	// clone the project
```

Comment:
just a thought, not on the changes in this PR - could we use a `-depth` here of some suitable number (say 50 or something to ensure we get all the relevant changes on the latest PRs), to save a bunch of un-necessary download?